src/views/HomeView.vue

```diff
@@ -1,10 +1,222 @@
+
+axios
+  .get('/sports')
+  .then(({ data }) => (sports.value = data))
+
+// A set of rows by sport
+// Within each sport there are is an array of Event objects
+const eventRows = computed(() => {
+  return sports.value?.map(sport => { 
+    const sportEvents = sport.eventIds.map(eventId => {
+      const event = events?.value?.find(event => event.id === eventId)
+
+      const [ teamOneData, teamTwoData ] = event.participants
+      const teamOneInfo = teams.value?.find(team => team.id === teamOneData.id) 
+      const teamTwoInfo = teams.value?.find(team => team.id === teamTwoData.id)
+
+      const [ teamOneSpreadId, teamOneMLId ] = teamOneData.betIds
+      const teamOneSpread = bets.value?.find(bet => bet.id === teamOneSpreadId).lineage
+      const teamOneML = bets.value?.find(bet => bet.id === teamOneMLId).lineage
+
+      const [ teamTwoSpreadId, teamTwoMLId ] = teamTwoData.betIds
+      const teamTwoSpread = bets.value?.find(bet => bet.id === teamTwoSpreadId).lineage
+      const teamTwoML = bets.value?.find(bet => bet.id === teamTwoMLId).lineage
+      
+      return {  
+        id: eventId,
+        firstTeam: {
+          name: teamOneInfo.name,
+          id: teamOneInfo.id,
+          spread: { id: teamOneSpreadId, 
+                    value: teamOneSpread },
+          ml: { id: teamOneMLId,
+                value: teamOneML } 
+        },
+        secondTeam: {
+          name: teamTwoInfo.name,
+          id: teamTwoInfo.id,
+          spread: { id: teamTwoSpreadId, 
+                    value: teamTwoSpread },
+          ml: { id: teamTwoMLId,
+                value: teamTwoML } 
+          },
+      }
+    })
+
+    return {
+      name: sport.name,
+      initial: sport.key,
+      allEvents: sportEvents
+    }
+  })
+})
+
+const getIcon = sportKey => {
+  switch (sportKey) {
+    case 'nhl':
+      return NhlIcon;
+    case 'mlb':
+      return MlbIcon;
+    case 'nfl':
+      return NflIcon;
+    case 'nba':
+      return NbaIcon;
+  }
+}
 
+const sportsWithIcon = computed(() => allSports.value.map(sport => ({
+  ...sport,
+  initials: sport.key.toUpperCase(),
+  icon: getIcon(sport.key)
+})))
+
+// // Submit button will have: 'eventId', 'betId', 'teamId'
+function sendBet(eventID, teamID, betObj){
+const bet = {
+        betId: betObj.id,
+        eventId: eventID,
+        teamId: teamID };
+
+axios.post('/submit', bet)
+  .then((result) => (console.log(result.data)))
+}
+</script>
 <template>
-  Homepage
+  <div class="container">
+    <Header>
+      <HeaderItem
+        v-for="sport in sportsWithIcon"
+        :key="sport.key"
+        :initials="sport.initials"
+        :icon="sport.icon"/>
+    </Header>
+
+    <aside class="sidebar">
+      Sidebar
+    </aside>
+
+<!-- TODO: Figure out how to make this less verbose -->
+    <main class="content">
+      <div v-for="sport in eventRows"
+      :key="sport.initial" 
+      class="sportRow">
+        <h2>{{ sport.name }}</h2>
```

Comment:
Just make a break here:

```
<div
   v-for="sport in eventRows"
```

And remember that if there are DOM tags on the same indented label to leave a space between them, just to make it more readable.

A suggestion about making it less verbose - you could always deconstruct your object in the v-for loop, something like

`v-for="({ key }) in arrayOfObjects"`

Reply:
Another suggestion, remember that v-slots can be dynamic by using `#[variable]`